### PR TITLE
Docs - Unify the two seprate readme files into a single file

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
- Fix broken example in readme
- Add links to production reference and documentation
- Remove an undeeded progress-report field

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
-- Easy to write, easy to read.
+// Lets create a new image
+const myImage = cld.image('sample');
+import {Resize} from '@cloudinary/base/actions/resize';
+myImage.resize(Resize.scale().width(100).height(100));
+
+// When we're done, we can apply all our changes and create a URL.
+const myURL = myImage.toURL();
+console.log(myURL);
+// https://res.cloudinary.com/demo/image/upload/c_scale,w_100,h_100/sample
```
Comment (carlevison) [inline README.md:42]:
```suggestion
// https://res.cloudinary.com/demo/image/upload/c_scale,h_100,w_100/sample
```

Comment (carlevison) [reply README.md:42]: The returned URL would have the transformations in alphabetical order, I believe.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,58 @@
...
+        secure: true // force https, set to false to force http
+    }
+});
 
-### Key Features
-- ESM packages, import only what you need
-- Declarative Syntax 
-- Fully typed for excellent IDE support
-- Easy to write, easy to read.
+// Lets create a new image
```
Comment (carlevison) [inline README.md:34]:
```suggestion
// Let's create a new image
```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,58 @@
...
-- <a href="https://cloudinary.com/blog/get_ready_for_cloudinary_s_next_generation_javascript_sdks">Cloudinary's next generation JS SDKs</a>
+## Installation
+```bash
+npm install @cloudinary/base 
+```
 
+## Simple usage
+```javascript
 
+// Import the cloudinary class
```
Comment (carlevison) [inline README.md:21]:
```suggestion
// Import the Cloudinary class
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,58 @@
-## Welcome
+# Welcome
 
-### About this project
-This project enables you to create Cloudinary URLs for your images.
-Using this SDK, you can apply advanced transformations to your images.
+## About this project
 
+This project enables you to create Cloudinary URLs for your images and videos.
+Using this SDK, you can apply advanced transformations to your images and videos.
```
Comment (carlevison) [inline README.md:7]: Might be worth adding that you can use this in conjunction with one of the front end frameworks and link to that repo too.

**Review by strausr — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (patrick-tolosa) [reply README.md:7]: Added that, and some extra links as well to Angular and React.

**Review by carlevison — APPROVED** [review]

**Pull request merged**